src/main/resources/configCivcraft.yml

```diff
@@ -7194,7 +7194,7 @@ recipes:
          - Compacted Item
       darkoaksaplings:
         material: SAPLING
-        amount: 32
+        amount: 16
```

Comment:
I think when making those recipes it was kinda forgotten how much harder dark oak is to farm compared to other saplings, so this definitely makes sense